### ISSUE TITLE
Add functionality to allow multiple of same companies to run in program.

### ISSUE THREAD

**Issue by KeeganShaw-GIS** [IM3S/LDAR_Sim#75]
Has dev_OGI been tested for programs that have both regular OGI requirements AND follow-up OGI? What happens? Does the same crew do both? Sounds like it could get complicated...

_Originally posted by @tarcadius in https://github.com/IM3S/LDAR_Sim/pull/67#discussion_r663586644_

**Comment by tbarchyn** [comment]
This is all about assignment of sites and scheduling? @AkaMrmuffin

**Comment by AkaMrmuffin** [comment]
Yes it’s can be done by using scheduling. If we are talking about same company working at same time. We are actually talking about multiple crews rather than companies. Multiple same tech companies working at the same time. For example, 2 aircraft companies doing same work together can be done by creating two program files. E.g. aircraft1{},aircraft2{}. Coleman actually did this type of simulations when he was simulating the AltFEMPs for AER.

**Comment by KeeganShaw-GIS** [comment]
I think this has been fixed in V2 by using labels to reference the method. will confirm.
